src/main/java/woohakdong/server/common/security/jwt/JWTFilter.java

```diff
@@ -0,0 +1,77 @@
+package woohakdong.server.common.security.jwt;
+
+import io.jsonwebtoken.ExpiredJwtException;
+import jakarta.servlet.FilterChain;
+import jakarta.servlet.ServletException;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.web.filter.OncePerRequestFilter;
+import woohakdong.server.api.controller.auth.dto.CustomUserDetails;
+import woohakdong.server.domain.member.Member;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+
+public class JWTFilter extends OncePerRequestFilter {
+    private final JWTUtil jwtUtil;
+
+    public JWTFilter(JWTUtil jwtUtil) {
+
+        this.jwtUtil = jwtUtil;
+    }
```

Comment:
이 부분을 Spring Container를 활용하여, 다음과 같이 수정할 수 있을 것 같습니다!

```suggestion
@Component
@RequiredArgsConstructor
public class JWTFilter extends OncePerRequestFilter {
    private final JWTUtil jwtUtil;
```